src/fixture/fixture.test.ts

```diff
@@ -68,3 +68,148 @@ test(`priotizes generators via extend`, () => {
+		string_ip: z.string().ip(),
+		string_ipv4: z.string().ip({ version: 'v4' }),
+		string_ipv6: z.string().ip({ version: 'v6' }),
+		string_trim: z.string().trim(),
+		toLowerCase: z.string().toLowerCase(),
+		toUpperCase: z.string().toUpperCase(),
+		number_gt: z.number().gt(5),
+		number_gte: z.number().gte(5),
+		number_lt: z.number().lt(5),
+		number_lte: z.number().lte(5),
+		number_int: z.number().int(),
+		number_positive: z.number().positive(),
+		number_nonnegative: z.number().nonnegative(),
+		number_negative: z.number().negative(),
+		number_nonpositive: z.number().nonpositive(),
+		number_multipleOf: z.number().multipleOf(5),
+		number_finite: z.number().finite(),
+		number_safe: z.number().safe(),
+		bigint_gt: z.bigint().gt(5n),
+		bigint_gte: z.bigint().gte(5n),
+		bigint_lt: z.bigint().lt(5n),
+		bigint_lte: z.bigint().lte(5n),
+		bigint_positive: z.bigint().positive(),
+		bigint_nonnegative: z.bigint().nonnegative(),
+		bigint_negative: z.bigint().negative(),
+		bigint_nonpositive: z.bigint().nonpositive(),
+		bigint_multipleOf: z.bigint().multipleOf(5n),
+		nan: z.nan(),
+		enum: z.enum(['Salmon', 'Tuna', 'Trout']),
+		native_enum: z.nativeEnum(Fruits),
+		string_enum: z.nativeEnum(StringFruits),
+		const_enum: z.nativeEnum(constFruits),
+		optional: z.optional(z.string()),
+		nullable: z.nullable(z.string()),
+		object: z.object({ name: z.string() }),
+		array: z.array(z.string()),
+		array_nonempty: z.string().array().nonempty(),
+		array_min: z.string().array().min(5),
+		array_max: z.string().array().max(5),
+		array_length: z.string().array().length(5),
+		tuples: z.tuple([
+			z.string(),
+			z.number(),
+			z.object({
+				pointsScored: z.number(),
+			}),
+		]),
+		unions: z.union([z.string(), z.number()]),
+		unions_or: z.string().or(z.number()),
+		union_discriminated: z.discriminatedUnion('status', [
+			z.object({ status: z.literal('success'), data: z.string() }),
+			z.object({ status: z.literal('failed'), error: z.instanceof(Error) }),
+		]),
+		record: z.record(z.number()),
+		record_keytype: z.record(z.string().min(1), z.number()),
+		map: z.map(z.string(), z.number()),
+		set: z.set(z.number()),
+		set_nonempty: z.set(z.number()).nonempty(),
+		set_min: z.set(z.number()).min(5),
+		set_max: z.set(z.number()).max(5),
+		set_size: z.set(z.number()).size(5),
+		intersection: z.intersection(Person, Employee),
+		intersection_and: Person.and(Employee),
+		promise: z.promise(z.number()),
+		function: z.function(),
+		custom: z.custom<`${number}px`>((val) => /^\d+px$/.test(val as string)),
+		instanceof: instanceOfSchema,
+	});
+
+	const fixture = new Fixture().extend(
+		Generator({
+			schema: instanceOfSchema,
+			output: () => new InstanceOfClass(),
+		})
+	);
+
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	const result = fixture.fromSchema(schemaWithEverything, { seed: 1 });
+	// TODO: this should parse
+	// schemaWithEverything.parse(result);
```

Comment:
Does this not currently parse?

Reply:
It currently fails on several properties.